.github/workflows/test_ert_with_slurm.yml

```diff
@@ -0,0 +1,76 @@
+on:
+  workflow_call:
+    inputs:
+      os:
+        type: string
+      python-version:
+        type: string
+jobs:
+  test-poly-on-slurm:
+    name: Run ert tests
+    timeout-minutes: 20
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - uses: actions/setup-python@v4
+      id: setup_python
+      with:
+        python-version: ${{ inputs.python-version }}
+        cache: "pip"
+        cache-dependency-path: |
+          setup.py
+          pyproject.toml
+
+    - name: Get wheels
+      uses: actions/download-artifact@v3
+      with:
+        name: ${{ inputs.os }} Python ${{ inputs.python-version }} wheel
+
+    - name: Install wheel
+      run: |
+        find . -name "*.whl" -exec pip install "{}[dev]" \;
+
+    - name: Install and setup slurm
+      run: |
+        set -e
+
+        sudo apt install slurmd slurmctld -y
+
+        cat <<EOF > slurm.conf
+        ClusterName=localcluster
+        SlurmctldHost=localhost
+        SelectType=select/cons_tres
+        SelectTypeParameters=CR_Core
+        ProctrackType=proctrack/linuxproc
+        PartitionName=LocalQ Nodes=ALL Default=YES MaxTime=INFINITE State=UP
+        EOF
+
+        # Self-configure the node:
+        slurmd -C | grep NodeName >> slurm.conf
+
+        cat slurm.conf
+
+        sudo mv slurm.conf /etc/slurm/
+        sudo systemctl start slurmd
+        sudo systemctl start slurmctld
+
+        sinfo
```

Comment:
Could you maybe put some comments into the code for those who don't know what does it entail to startup slurm? (like me). Thanks!

Reply:
Fixed